ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -113,7 +113,7 @@ export default function BuildQuote({
   const toTokenIsNotEth =
     selectedToToken?.address &&
     selectedToToken?.address !== ETH_SWAPS_TOKEN_OBJECT.address;
-
+  const occurances = Number(0);
```

Comment:
assuming Number(0) was used for testing, should probably be `Number(selectedToToken.occurances)`

Reply:
Yep, I had set for testing -- my apologies.